main.py

```diff
@@ -1,7 +1,28 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genre = ["Western", "Action", "Dramma"]
+
+    for genre in genre:
+        Genre.objects.create(name=genre)
+
+    first_name = ["George", "Kianu", "Scarlett", "Will", "Jaden", "Scarlett"]
+    last_name = ["Klooney", "Reaves", "Keegan", "Smith", "Smith", "Johansson"]
```

Comment:
You can create a list with tuple [('first_name', 'last_name')] and iterate without zip